src/api/spec/features/webui/requests_spec.rb

```diff
@@ -163,7 +163,16 @@
 
   context 'review' do
     describe 'for user' do
-      skip
+      let(:reviewer) { create(:confirmed_user) }
```

Comment:
Then is ok, before we started doing this... but if it have a sequence, it is ok I guess :smile: